### DISCUSSION TITLE
Is it possible to run my app on a device with live-reload?

### DISCUSSION THREAD

**Discussion by BerkeAras** [ionic-team/capacitor#3102] [General] [answered]
Hello!

Is it possible to run my app on a device (in android studio) with live-reload? In cordova the command is `cordova run <platform>`, but in capacitor?

Thanks.

**Comment by imhoffd** [comment, marked as answer]
Live-reload is achieved with several tools working together. Capacitor does its part by offering a way to set the Web View URL ([see the docs](https://capacitor.ionicframework.com/docs/basics/configuring-your-app#common-configuration)), but you need an actual server running your app. Many Framework CLIs offer this capability, but don't integrate with Capacitor. That's where the Ionic CLI can be helpful, because it offers `ionic cap run <platform> --livereload` ([see the live-reload docs for Ionic](https://ionicframework.com/docs/cli/livereload)).

`cordova run <platform>` doesn't _quite_ get you all the way because you still need a dev server running, too. If you're asking for a way to build and deploy your app to a device in the terminal, Capacitor doesn't have that feature, but stay tuned... 😉

**Comment by dotNetkow** [comment]
As dwieeb mentioned, stay tuned... in the meantime we've added a [new Live Reload guide](https://capacitorjs.com/docs/guides/live-reload) in the Capacitor docs.
